collectors/python.d.plugin/isc_dhcpd/isc_dhcpd.chart.py

```diff
@@ -86,6 +86,42 @@ class Pool:
     def __init__(self, name, network):
         self.id = re.sub(r'[:/.-]+', '_', name)
         self.name = name
+
+        self.networks = list()
+        for network in network.split(" "):
+            if not network:
+                continue
+
+            ip_type = self.detect_ip_type(ip=network)
+            if ip_type == "CIDR":
+                self.networks.append(PoolCIDR(network=network))
+            elif ip_type == "IP_RANGE":
+                self.networks.append(PoolIPRange(ip_range=network))
+            else:
+                raise ValueError('Network ({0}) syntaxis error!'.format(network))
+
+    @staticmethod
+    def detect_ip_type(ip):
+        ip_type = ip.split("-")
+        if len(ip_type) == 1:
+            return "CIDR"
+        elif len(ip_type) == 2:
+            return "IP_RANGE"
+        else:
+            return "UNKNOWN"
```

Comment:
lets make it a function, not class method

"CIDR", "IP_RANGE", "UNKNOWN" should be global vars

```python
POOL_CIDR = "CIDR"
POOL_IP_RANGE = "IP_RANGE"
POOL_UNKNOWN = "UNKNOWN"
```